app/models/user.js

```diff
@@ -21,5 +24,9 @@ export default Addressable.extend({
 
   fullName: Ember.computed('firstName', 'lastName', function(){
     return (this.get('firstName') + " " + this.get('lastName'));
+  }),
+
+  isInfoComplete: Ember.computed('firstName', 'lastName', 'email', 'mobile', function(){
+    return this.get('firstName') && this.get('firstName').length && this.get('lastName') && this.get('lastName').length && this.get('email') && this.get('email').length && this.get('mobile') && this.get('mobile').length && this.get('title') && this.get('title').length;
```

Comment:
Can we refactor this??
May be write a function which checks values for attributes exists or not and returns true or false??
May be something like this would work

```
hasValue(...args) {
  args.forEach(arg => {
    if(!this.get("arg") || !this.get("arg").length) {
      return false;
    }
  })
}
```